# Polar velocity message for interfacing with the Neato.
float64 linear
float64 angular
